### ISSUE TITLE
Intermittent UI test failure - < ContextMenusTest. verifyContextOpenImageNewTab >

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#24420]
### Firebase Test Run:
[Firebase link](https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/6481699293202036185/executions/bs.55a3250d1e8d13b0/testcases/1/test-cases)
[Firebase link](https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/6481699293202036185/executions/bs.b6974a091eb4918b/testcases/1/test-cases)

### Stacktrace:
[Logcat](https://github.com/mozilla-mobile/fenix/files/8334491/verifyContextOpenImageNewTab.txt)
[Logcat](https://github.com/mozilla-mobile/fenix/files/8334492/verifyContextOpenImageNewTab2.txt)

**Comment by AndiAJ** [comment]
Failed 1x on 3/23 Main 
[Firebase link](https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/7772509306765377746/executions/bs.bf1d8107649a251c/testcases/1/test-cases)
[Firebase link](https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/7772509306765377746/executions/bs.526fb5ca21472427/testcases/1/test-cases)

**Comment by sv-ohorvath** [comment]
A11y issue causing the failure was fixed, can be reopened if it fails again.
